registry.json

```diff
@@ -3303,12 +3318,13 @@
       "reddit": "jangid",
       "instagram": "",
       "youtube": ""
-    },
+   },
+   {
```

Comment:
I believe this is now unnecessary - im not sure what has happened here... Are you able to validate your copy of the json as just removing this `{` should be enough to correct the json?